<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenNetlistView::DialogAbout</class>
 <widget class="QDialog" name="OpenNetlistView::DialogAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About OpenNetlistView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="constLVersion">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lVersion">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lHeading">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>OpenNetlistView</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lInformation">
     <property name="text">
      <string>OpenNetlistView is an interactive schematic viewer for Yosys netlists
in the JSON format.
The software is licenced under GPL3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="constLContrib">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Contributurs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lContrib">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pAboutQt">
       <property name="text">
        <string>About Qt ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="pDialogButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pDialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>OpenNetlistView::DialogAbout</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pDialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>OpenNetlistView::DialogAbout</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
